modules/acquia_cms_common/acquia_cms_common.module

```diff
@@ -336,4 +322,17 @@ function acquia_cms_common_modules_installed($modules) {
       ->set('toolkit', 'imagemagick')
       ->save();
   }
+  $acms_modules = [
+    'acquia_cms_article',
+    'acquia_cms_event',
+    'acquia_cms_image',
+    'acquia_cms_page',
+    'acquia_cms_person',
+    'acquia_cms_place',
+    'acquia_cms_search',
+    'acquia_cms_site_studio',
+  ];
+  if (array_intersect($acms_modules, $modules) && (PHP_SAPI === 'cli')) {
+    \Drupal::service('acquia_cms_common.utility')->siteStudioPackageImport();
```

Comment:
@rajeshreeputra What are we trying to do here ?

Reply:
We are trying to import packages related to ACMS module with cohesion_sync if any of this module gets installed and it should rebuild the same